src/GraphQL.Client/GraphQLHttpClient.cs

```diff
@@ -127,32 +127,30 @@ public IObservable<GraphQLResponse<TResponse>> CreateSubscriptionStream<TRespons
         private async Task<GraphQLHttpResponse<TResponse>> SendHttpPostRequestAsync<TResponse>(GraphQLRequest request, CancellationToken cancellationToken = default)
```

Comment:
Once you delegated the creation of the `httpRequestMessage` to the caller, this method is no longer `SendHttpPOSTRequestAsync`. It is just `SendHttpRequestAsync` because I can send a GET request in the format defined in [http spec](https://github.com/APIs-guru/graphql-over-http#get).